azure-pipelines.yml

```diff
@@ -19,6 +19,11 @@ jobs:
       name: linux
       vmImage: ubuntu-18.04
       matrix:
+        amd64-linux-py10:
+          MB_PYTHON_VERSION: "3.10.0"
+        32bit-amd64-linux-py10:
+          MB_PYTHON_VERSION: "3.10.0"
+          PLAT: "i686"
```

Comment:
```suggestion
        amd64-linux-py10:
          AZURE_PYTHON_VERSION: "3.9"
          MB_PYTHON_VERSION: "3.10.0"
        32bit-amd64-linux-py10:
          AZURE_PYTHON_VERSION: "3.9"
          MB_PYTHON_VERSION: "3.10.0"
          PLAT: "i686"
```

Reply:
```
AZURE_PYTHON_VERSION: "3.9"
MB_PYTHON_VERSION: "3.10.0"
DOCKER_TEST_IMAGE: "multibuild/focal_{PLAT}"  # this as well?
```
Focal is needed to get python 3.10 but only has 64 bit versions, so no 32 bit wheels. I used `"3.10"` for the version, guess we will see if `"3.10.0"` works as well :) NumPy is going to drop 32 bit manylinux wheels for 3.10 and only keep them for Windows.

Reply:
Ok so I should remove the manylinux 32 bit for 3.10?